applications/GeoMechanicsApplication/tests/test_transient_thermal.py

```diff
@@ -0,0 +1,157 @@
+import os
+
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import test_helper
+
+class KratosGeoMechanicsTransientThermalTests(KratosUnittest.TestCase):
+    """
+    This class contains benchmark tests which are checked with the analytical solution
+    """
+
+    def setUp(self):
+        # Code here will be placed BEFORE every test in this TestCase.
+        pass
+
+    def tearDown(self):
+        # Code here will be placed AFTER every test in this TestCase.
+        pass
+    
```

Comment:
It seems [Codacy ](https://app.codacy.com/gh/KratosMultiphysics/Kratos/pullRequest?prid=13005264)is complaining about a trailing whitespace here:
![image](https://github.com/KratosMultiphysics/Kratos/assets/56549273/7b8b09f1-6b24-4e22-a7bb-d8d1716ab831)